src/qt/optionsmodel.cpp

```diff
@@ -526,7 +526,7 @@ bool OptionsModel::setOption(OptionID option, const QVariant& value, const std::
     case MapPortNatpmp: // core option - can be changed on-the-fly
         if (changed()) {
             update(value.toBool());
-            node().mapPort(value.toBool());
+            if (value.toBool()) node().mapPort();
```

Comment:
By adding this if here, this no longer accounts for the situation where port mapping is turned off. i think i slightly prefer the old API.

Reply:
Ah, yeah, right, that was stupid. I'll stick to just renaming `use_pcp` to `enabled`.

Reply:
Done.